whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -4404,6 +4404,7 @@
         "TODO:definedElsewhereToken": "7"
       },
       "i2": {"marcDefault": " ", "TODO": "ok to lock as undefined?"},
+      "$d": {"property": "code"},
```

Comment:
Since `code` is on par with `value` (akin to `label`), we need something "additional".

I suppose `note` made sense, apart from it not being a literal BF2 property, and apart from us attempting to avoid it (to not confuse it with `hasNote`).

Reply:
Since we have used comment for all literal notes we use this consequently here as well.